packages/promotion/src/models/promotion-rule-value.ts

```diff
@@ -0,0 +1,37 @@
+import {
+  BeforeCreate,
+  Entity,
+  ManyToOne,
+  OnInit,
+  PrimaryKey,
+  Property,
+} from "@mikro-orm/core"
+
+import { generateEntityId } from "@medusajs/utils"
+import PromotionRule from "./promotion-rule"
+
+@Entity()
+export default class PromotionRuleValue {
+  @PrimaryKey({ columnType: "text" })
+  id!: string
+
+  @ManyToOne(() => PromotionRule, {
+    onDelete: "cascade",
+    fieldName: "promotion_rule_id",
+    index: "IDX_promotion_rule_promotion_rule_value_id",
+  })
+  promotion_rule: PromotionRule
```

Comment:
question: Shouldn't the model include the `fieldName` property? Here, `promotion_rule_id`.

Reply:
its not needed for the relationship to function for nested create/update. Is there a reason for it to exist?

Reply:
No, not really – I was just curious about if this will change anything. Hadn't noticed it before in other modules